### PR TITLE
Add picture and text about protocol. Add local links for pictures. Fi…

### PR DESCRIPTION
**Pull request by jovirkku**
…x some typos. Rephrase here and there.

There was a Forum post where the writer had failed in finding how to select the IP protocol for mobile connections. This triggered me to add one more screenshot and some words. I also used the opportunity to add local links for the pictures to make it easier to point to them. A couple of typos were fixed, too.

### CODE REVIEW

File: Support/Help_Articles/Internet_Connections/README.md [hunk 1]
```diff
@@ -63,38 +63,43 @@ There is a corresponding selection ("Use SIM card") for mobile data in the menu
...
+6.  Although Sailfish OS is supposed to automatically get and set the appropriate settings for your connections, it is also possible to manually adjust them at the **"Data access point"** (see **[Pic 4](#pic_4)**). Please consult your mobile operator/carrier about the correct settings. In particular, check what kind of IP connections are supported, i.e., if the IP protocol is IPv4 or IPv6. If the operator still supports IPv4, we would recommend using that (instead of IPv6) as there may still be problems with IPv6 in some cases. On the other hand, the **[CLAT service](https://forum.sailfishos.org/t/14752#improving-mobile-data-connectivity-with-the-community-4)** added to OS release 4.5.0 should help using IPv6.
 
 <div class="flex-images" markdown="1">
 
-* <a href="MD-not-conn.png" class="narrow-image"><img src="MD-not-conn.png" alt="Options for mobile data"></a>
+* <a href="MD-not-conn.png" name="pic_3"><img src="MD-not-conn.png" alt="Options for mobile data"></a>
   <span class="md_figcaption">
     Pic 3: Options for mobile data
   </span>
+* <a href="DAP.png" name="pic_4"><img src="DAP.png" alt="Data access point settings"></a>
```
Comment (vigejolla) [inline Support/Help_Articles/Internet_Connections/README.md:81]: DAP.png is missing from the PR

**Review by vigejolla — CHANGES_REQUESTED** [review]
One missing image, otherwise LGTM

**Review by jpwalden — APPROVED** [review]
Agree with Ville, content looks good to me.

File: Support/Help_Articles/Internet_Connections/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Help_Articles/Internet_Connections/README.md:81]: My bad. I reused the path/file combo from the editor for the commit, so it had the README only. The new picture was in my work directory, so Jekyll found it.

**Review by vigejolla — APPROVED** [review]

**Pull request merged**